floodviz/static/js/map.js

```diff
@@ -100,45 +99,66 @@
 			add_paths(options.rivers_data, "river");
 			add_circles(options.ref_data, "ref-point", 2);
 			// Add sites and bind events for site hovers
-			var sites = add_circles(options.site_data, "gage-point", 3);
+			var sites = add_circles(options.site_data, "gage-point", 3, 'id');
 			sites.selectAll("circle")
-				.on('mousemove', function(d) {return self.mousemove(d.properties.name, d.properties.id)})
-				.on("mouseout", function() {return self.mouseout()});
+				.on('mousemove', function (d) {return self.mousemove(d.properties.name, d.properties.id)})
+				.on("mouseout", function (d) {return self.mouseout(d.properties.id)})
+				.on('click', function (d) { return self.click(d.properties.id)});
 			// Debug points
 			if (FV.mapinfo.debug) {
```

Comment:
I wonder if we should just but debug on the FV directly. I think I would like to see config variables in general in there own property. Something like FV.CONFIG = {debug: {{config.DEBUG}}, etc. It's really not mapinfo specific and we may want to add debug features to other parts of the application. Something to consider.